### PR TITLE
Fixes #10446, shorten "out of element" message

### LINKED ISSUE #10446: Shorten "out of element" messageShortening "out of element" message
### Is your feature request related to a problem? Please describe.
In browse mode or documents, when leaving a particular element with the caret, "out of element" is announced.

### Describe the solution you'd like
Based on how this is implemented in braille, I'd like to suggest "element end". So, instead of "out of list", announce 'list end". This is somewhat shorter and therefore more intuitive. Furthermore, the change is relatively small.

### Describe alternatives you've considered
Suppress element ends if they are followed by an element of the same role, see #10420

### PR DESCRIPTION
**Pull request by yozaam**
Based on the suggestion this has been made into  "element end"

### Link to issue number:
 #10446 

### Summary of the issue:
Instead of "out of list", announce 'list end", since this is shorter

### Description of how this pull request fixes the issue: 
Have changed the strings where we have the longer message

### Change log entry:

Section: New features, Changes, Bug fixes

### CODE REVIEW

**Comment by leonardder** [comment]
@yozaam Out of interest, how did you manage in asking me for review? I thought this was only for collaborators.

**Comment by yozaam** [comment]
> @yozaam Out of interest, how did you manage in asking me for review? I thought this was only for collaborators.

I'm sorry, when sending the request I just got the button on the right to ask for a reviewer, and since you opened the issue I though I'll ask you

**Comment by leonardder** [comment]
> I'm sorry, when sending the request I just got the button on the right to ask for a reviewer, and since you opened the issue I thought I'll ask you

No problem at all. I'm just amazed that you were able to see this button.

File: source/speech/__init__.py [hunk 1]
```diff
@@ -1696,7 +1696,7 @@ def getControlFieldSpeech(  # noqa: C901
 			joinedRoleText = " ".join(roleTextSequence)
 			out = [
 				# Translators: Indicates end of something (example output: at the end of a list, speaks out of list).
```
Comment (leonardder) [inline source/speech/__init__.py]: Please fix this comment accordingly

**Review by LeonarddeR — CHANGES_REQUESTED** [review]
There are several additional `out of` messages, please use `git grep "_(\"out of "` to find them

File: source/speech/__init__.py (hunk 1 above)
Comment (yozaam) [reply source/speech/__init__.py]: Yes I will do this ASAP, thank you

**Comment by Adriani90** [comment]
@yozaam you should fix the flake8 errors otherwise this PR cannot be merged.
1. Go to test results
2. Go to artefacts
3. Open flake8errors.txt where the errors are displayed along with the source file
4. solve them in the source file.
In your case it seems there are just white space errors before and after the = operator in a function.

**Comment by yozaam** [comment]
> @yozaam you should fix the flake8 errors otherwise this PR cannot be merged.
> 
>     1. Go to test results
> 
>     2. Go to artefacts
> 
>     3. Open flake8errors.txt where the errors are displayed along with the source file
> 
>     4. solve them in the source file.
>        In your case it seems there are just white space errors before and after the = operator in a function.

Thank you so much for explaining it to me in this format :D I am fixing it right now

**Review by feerrenrut — CHANGES_REQUESTED** [review]
Apologies this has been pending for so long, and welcome to the NVDA project 🎉

As I [mentioned on the issue](https://github.com/nvaccess/nvda/issues/10446#issuecomment-773852041) I'm happy let our alpha users try this out and give feedback. 

Though using `git grep -i "_(\"out of" -- "*.py"` I found one more:
`source/appModules/kindle.py:                _("out of popular highlight")`

Fix this up and we can go ahead.

**Comment by CyrilleB79** [comment]
Hello
Sorry to comment here a bit late.

Here is a strange use case that we should take into consideration.
Steps to reproduce:
* On a web page, press L to go to first element of a list
* Press UpArrow
In current version of NVDA, you hear "Out of list".
With this PR, if I understand correctly, you would hear "List end". This seems strange because we go out of the list across its start boundary.

Any feedback from braille users on this point?

For real containers (tables, lists, etc.), I do not have any better suggestion than the existing "out of". For containers that seem more formatting (e.g. spelling error, link), I guess "no spelling error" / "no link" would be suitable, but maybe it's out of the scope of this PR.

I do not advocate for any specific solution or change. But I wanted at least highlight this use case.

I do not

**Comment by feerrenrut** [comment]
Thanks @CyrilleB79, that's a very useful observation. I'm not sure what to suggest either. I'm also interested in braille users perspective on this. For those not familiar with the braille behavior the start of a list shows `lst` and the end shows `lst end`. If going through the document backwards, you encounter `lst end` first then `lst`.

**Comment by seanbudd** [comment]
Closed as this has been taken over by the more recently maintained and created #12260

**Pull request closed without merging**